manuscript/revision_response.Rmd

```diff
@@ -0,0 +1,113 @@
+
+The reviewer here raises a very good point here about employing domain knowledge to reduce computational costs. This is something that we alluded to here,
+
+"Fully detailed conservation decision-making problems will likely require comparable specialized algorithms and hardware that ecologists do not generally have access to."
+
+
+
+Reviewer 3:
+
+1) I strongly recommend that the authors rewrite portions of the manuscript to make it accessible to a broader audience unfamiliar with deep learning techniques.
+
+We acknowledge that our paper might be difficult to grasp for the audience of MEE, but we want to underscore that deep RL is difficult to understand in general, even for people that are familiar with the other subfields of machine learning. There are a lot of concepts and terminology that are unique to RL that will likely require newcomers to do lots of re-reading and digging through the references to gain a comprehensive understanding. This being said, we believe we’ve made subject matter as accessible as possible by explaining the fundamentals in the sections “RL Environments”  and “Deep RL Agents” without any hard-to-parse equations and minimal jargon. In the appendices, we go a bit more in-depth on some theoretical and practical topics, which will hopefully guide readers that are motivated to dig deeper to reach a better understanding of the subject. Lastly, on this area of concern, we have addressed some specific areas of confusion raised by other reviewers.
+
+2) While RL is based on mathematical and probabilistic outcomes proposing actions to maximize profit, I wonder whether this shall provide feasible outcomes as the actions may only be mathematically attainable and not ecologically. Suppose this is being done to reduce human interventions at decision making, to identify and obtain viable strategies, one has to investigate the outputs of RL strategies. I suggest the authors consider discussing this briefly in the manuscript.
+
+
+We interpret this comment to be partially asking the question: “What are the limitations on an action space?” We have changed some of the language in “RL Environments” to better address this question:
```

Comment:
I think we should reword: 

> The reviewer raises two important issues here:  "What is the appropriate reward function", and "What are the appropriate limitations on the action space".  These questions have always been central issues to ecological management and decision-making, and are neither unique to or resolved by the use RL. 

can cite some textbook examples: https://onlinelibrary.wiley.com/doi/book/10.1002/9781444398557, https://onlinelibrary.wiley.com/doi/book/10.1002/9781118506196